<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddReportLossDlg</class>
 <widget class="QDialog" name="AddReportLossDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>46</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{
	background-color:#505050;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>284</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>284</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="saveTopWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>64</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#saveTopWidget{
	background-color:#fafafa;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>18</number>
            </property>
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QLabel" name="goodsNameLabel">
              <property name="text">
               <string>商品名称：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="goodsCodeLabel">
              <property name="text">
               <string>商品编号：</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="saveMainWidget" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget#saveMainWidget{
	background-color:#ffffff;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="lossNoLabel">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>报损数量:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lossNoLineEdit">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/shareImages/editRect.png);
padding-left:10px;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="lollReasonLabel">
                <property name="maximumSize">
                 <size>
                  <width>54</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>报损原因:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="outofdateBtn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QRadioButton:indicator{
   width:22px;
   height:0px;

}
QRadioButton{
border:1px groove &quot;#666666&quot;;
border-radius:6px;
padding:2px 4px;
}

QRadioButton:checked{
border:2px groove &quot;#EA8852&quot;;
border-radius:6px;
padding:2px 4px;
}</string>
                </property>
                <property name="text">
                 <string>过期</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="badBtn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QRadioButton:indicator{
   width:22px;
   height:0px;

}
QRadioButton{
border:1px groove &quot;#666666&quot;;
border-radius:6px;
padding:2px 4px;
}

QRadioButton:checked{
border:2px groove &quot;#EA8852&quot;;
border-radius:6px;
padding:2px 4px;
}</string>
                </property>
                <property name="text">
                 <string>损坏</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>54</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QRadioButton" name="lossBtn">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QRadioButton:indicator{
   width:22px;
   height:0px;

}
QRadioButton{
border:1px groove &quot;#666666&quot;;
border-radius:6px;
padding:2px 4px;
}

QRadioButton:checked{
border:2px groove &quot;#EA8852&quot;;
border-radius:6px;
padding:2px 4px;
}</string>
                </property>
                <property name="text">
                 <string>丢失</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="anotherBtn">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QRadioButton:indicator{
   width:22px;
   height:0px;

}
QRadioButton{
border:1px groove &quot;#666666&quot;;
border-radius:6px;
padding:2px 4px;
}

QRadioButton:checked{
border:2px groove &quot;#EA8852&quot;;
border-radius:6px;
padding:2px 4px;
}</string>
                </property>
                <property name="text">
                 <string>其他</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="hintLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>18</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>18</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:red;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>42</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>42</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
color:#ffffff;
font-size:16px;
border-image: url(:/loginImage/commonButton.png);
}

QPushButton:pressed{
border-image: url(:/loginImage/commonButtonPressed.png);
}</string>
              </property>
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="WndDigitalKeyboard" name="keyboardWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>304</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>304</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WndDigitalKeyboard</class>
   <extends>QWidget</extends>
   <header>WndDigitalKeyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
